### ISSUE TITLE
"VPC Support" section needs update now that Serverless VPC connector are launched 

### ISSUE THREAD

**Issue by steren** [ahmetb/cloud-run-faq#78]

**Comment by ahmetb** [comment]
@steren It's actually up-to-date I think? Any problem with this:

```
### Can I place my Cloud Run application inside a VPC network?

Currently no (see [here](#can-i-run-cloud-run-applications-on-a-private-ip));
however, you can make requests to other resources inside the VPC network using
the "VPC Access Connector" (see next question).

### How to connect IPs in a VPC network from Cloud Run?

Cloud Run now has beta support for "VPC Access Connector". This feature allows
Cloud Run applications to be able to connect private IPs in the VPC (but not the
other way).
```

**Comment by steren** [comment]
Ah I see. The "currently no" is misleading, as the answer is "sort of" with vpc connectors. I suggest to rephrase

**Comment by ahmetb** [comment]
Fixed in eb923ca if you'd like to double-check.
